libraries/src/test/java/com/baeldung/mapdb/HelloBaeldungUnitTest.java

```diff
@@ -0,0 +1,51 @@
+package com.baeldung.mapdb;
+
+import org.junit.Test;
+import org.mapdb.DB;
+import org.mapdb.DBMaker;
+import org.mapdb.HTreeMap;
+
+import java.util.concurrent.ConcurrentMap;
+
+import static junit.framework.Assert.assertEquals;
+
+public class HelloBaeldungUnitTest {
+
+  @Test
+  public void givenInMemoryDBInstantiateCorrectly_whenDataSavedAndRetrieved_checkRetrievalCorrect() {
+
+    DB db = DBMaker.memoryDB().make();
+
+    final String WELCOME_MESSAGE_KEY = "Welcome Message";
+    final String WELCOME_MESSAGE = "Hello Baeldung!";
```

Comment:
Similar feedback relating to `final` here too

Reply:
Same as above